packages/updating-element/src/updating-element.ts

```diff
@@ -779,6 +847,28 @@ export abstract class UpdatingElement extends HTMLElement {
     if (!this.isUpdatePending) {
       return;
     }
+    // create renderRoot before first update.
+    if (!this.hasUpdated) {
+      // Produce warning if any class properties are shadowed by class fields
+      if (DEV_MODE) {
+        (this.constructor as typeof UpdatingElement).elementProperties!.forEach(
+          (_v, p) => {
+            if (this.hasOwnProperty(p) && !this._instanceProperties?.has(p)) {
+              console.warn(
+                `Reactive properties cannot be initialized using class ` +
```

Comment:
I'd change this warning text since in source text you _can_ initialize class fields with initializers depending on decorators and compiler settings. You mention decorators, but then those sometimes don't work depending on the compiler. I would definitely link to docs from this message. Want to chat to brainstorm the message?

Reply:
Updated message.